### ISSUE TITLE
Username and Password in script.config may be too insecure

### ISSUE THREAD

**Issue by jarodburchill** [jarodburchill/ProjectAutomation#11]
We need to find a good way of securing this data and make sure it never ends up in git history. Alternatively, we just make users enter their password no matter what.

**Comment by ajnieset** [comment]
we could just make it so that it reads in environment variables via os. This puts the onus on the user to set environment variables but is generally a secure practice

**Comment by Rexogamer** [comment]
would a gitignore suffice?

**Comment by ajnieset** [comment]
i dont think we want to ignore the whole file. just the username and password

**Comment by Rexogamer** [comment]
maybe we’d make a separate file for the username/pass (eg script2.config) and .gitignore that
